AppCenterAuth/AppCenterAuth/Internals/Models/MSAADAuthority.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+#import <MSAuthority.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSAADAuthority : MSAuthority {
```

Comment:
You don't need `{}`s for interfaces, just the `@end` marker

Reply:
yeah. didn't realize. :)